src/main/java/carrierManagementSystem/RfPorts.java

```diff
@@ -0,0 +1,27 @@
+package carrierManagementSystem;
+
+/**
+ * This class contain information related to rf ports.
+ */
+public enum RfPorts {
```

Comment:
Same comment about singular vs plural

Reply:
I guess it should be plural as we configure several ports for both RAT type.